pkgs/development/libraries/ocl-icd/default.nix

```diff
@@ -1,15 +1,25 @@
-{lib, stdenv, fetchurl, ruby, opencl-headers, addOpenGLRunpath }:
+{ lib
+, stdenv
+, fetchurl
+, ruby
+, opencl-headers
+, addOpenGLRunpath
+, autoreconfHook
+}:
 
 stdenv.mkDerivation rec {
   pname = "ocl-icd";
-  version = "2.2.10";
+  version = "2.2.13";
```

Comment:
2.3.1 is out, any reason to not use the latest version?

Reply:
I wanted to cheat a bit and rely on the bot for automatic update once we point it an up to date repo :) Split commit in two: switch homepage without release increment and updated release separately (in unlikely case of the needed revert).